models/staging/schema.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
```

Comment:
nice nice! feel free to add tests for the other two staging models (also to your dim tables!)